spin/spin.go

```diff
@@ -65,15 +71,48 @@ func commandStart(command []string) tea.Cmd {
 		}
 
 		executing = exec.Command(command[0], args...) //nolint:gosec
-		if term.IsTerminal(os.Stdout.Fd()) {
+		executing.Stdin = os.Stdin
+
+		isTerminal := term.IsTerminal(os.Stdout.Fd())
+
+		//nolint:nestif
+		if isTerminal && runtime.GOOS == "windows" {
 			executing.Stdout = io.MultiWriter(&bothbuf, &outbuf)
 			executing.Stderr = io.MultiWriter(&bothbuf, &errbuf)
+			_ = executing.Run()
```

Comment:
After some headache, me and @aymanbagabas decided to keep the old behavior on Windows for now.

(Git Bash was freezing for some reason, although the other terminals worked correctly).

We can reconsider in the future if we find a reliable way to make it work on Windows.